exporter/datadogexporter/metrics_exporter.go

```diff
@@ -178,6 +178,7 @@ func (exp *metricsExporter) PushMetricsData(ctx context.Context, md pdata.Metric
 
 	err = nil
 	if len(ms) > 0 {
+                exp.params.Logger.Debug("exporting payload", zap.Any("metric", ms))
```

Comment:
Indentation here looks odd. Did you run gofmt and/or the linter target? This will likely fail the CI.

Reply:
@jpkrohling Can you please review this PR again. Tested with gofmt and linter this time.